### PR TITLE
Allow configuring default options for admin service

### PR DESCRIPTION
**Pull request by kunicmarko20**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Default admin service options (group, label catalogue and icon) are now configurable

```

### CODE REVIEW

File: src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php [hunk 1]
```diff
@@ -85,9 +85,9 @@ public function process(ContainerBuilder $container)
                     continue;
                 }
 
-                $resolvedGroupName = isset($attributes['group']) ? $parameterBag->resolveValue($attributes['group']) : 'default';
-                $labelCatalogue = isset($attributes['label_catalogue']) ? $attributes['label_catalogue'] : 'SonataAdminBundle';
-                $icon = isset($attributes['icon']) ? $attributes['icon'] : '<i class="fa fa-folder"></i>';
+                $resolvedGroupName = isset($attributes['group']) ? $parameterBag->resolveValue($attributes['group']) : $container->getParameter('sonata.admin.configuration.default_group');
```
Comment (greg0ire) [inline src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php]:
```suggestion
                $resolvedGroupName = isset($attributes['group']) ? 
                    $parameterBag->resolveValue($attributes['group']) :
                    $container->getParameter('sonata.admin.configuration.default_group');
```

File: src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php [hunk 2]
```diff
@@ -85,9 +85,9 @@ public function process(ContainerBuilder $container)
...
+                $labelCatalogue = isset($attributes['label_catalogue']) ? $attributes['label_catalogue'] : $container->getParameter('sonata.admin.configuration.default_label_catalogue');
```
Comment (greg0ire) [inline src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php]:
```suggestion
                $labelCatalogue = isset($attributes['label_catalogue']) ?
                    $attributes['label_catalogue'] :
                    $container->getParameter('sonata.admin.configuration.default_label_catalogue');
```

File: src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php [hunk 3]
```diff
@@ -85,9 +85,9 @@ public function process(ContainerBuilder $container)
...
+                $icon = isset($attributes['icon']) ? $attributes['icon'] : $container->getParameter('sonata.admin.configuration.default_icon');
```
Comment (greg0ire) [inline src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php]:
```suggestion
                $icon = isset($attributes['icon']) ?
                    $attributes['icon'] :
                    $container->getParameter('sonata.admin.configuration.default_icon');
```

File: src/DependencyInjection/Configuration.php [hunk 4]
```diff
@@ -127,6 +127,9 @@ public function getConfigTreeBuilder()
                         ->booleanNode('use_stickyforms')->defaultTrue()->end()
                         ->integerNode('pager_links')->defaultNull()->end()
                         ->scalarNode('form_type')->defaultValue('standard')->end()
+                        ->scalarNode('default_group')->defaultValue('default')->end()
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]: Please add `info` calls for every node you added.

File: src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php (hunk 1 above)
Comment (kunicmarko20) [reply src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php]: was lazy :sweat_smile: will do it now

File: src/DependencyInjection/Configuration.php (hunk 4 above)
Comment (kunicmarko20) [reply src/DependencyInjection/Configuration.php]: added

File: src/DependencyInjection/Configuration.php [hunk 5]
```diff
@@ -127,6 +127,9 @@ public function getConfigTreeBuilder()
                         ->booleanNode('use_stickyforms')->defaultTrue()->end()
                         ->integerNode('pager_links')->defaultNull()->end()
                         ->scalarNode('form_type')->defaultValue('standard')->end()
+                        ->scalarNode('default_group')->defaultValue('default')->info("Group used for admin services if one isn't provided.")->end()
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]:
```suggestion
                        ->scalarNode('default_group')
                            ->defaultValue('default')
                            ->info("Group used for admin services if one isn't provided.")
                        ->end()
```

File: src/DependencyInjection/Configuration.php [hunk 6]
```diff
@@ -127,6 +127,9 @@ public function getConfigTreeBuilder()
...
+                        ->scalarNode('default_label_catalogue')->defaultValue('SonataAdminBundle')->info("Label Catalogue used for admin services if one isn't provided.")->end()
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]:
```suggestion
                        ->scalarNode('default_label_catalogue')
                            ->defaultValue('SonataAdminBundle')
                            ->info('Label Catalogue used for admin services if one is not provided.')
                        ->end()
```

File: src/DependencyInjection/Configuration.php [hunk 7]
```diff
@@ -127,6 +127,9 @@ public function getConfigTreeBuilder()
...
+                        ->scalarNode('default_icon')->defaultValue('<i class="fa fa-folder"></i>')->info("Icon used for admin services if one isn't provided.")->end()
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]:
```suggestion
                        ->scalarNode('default_icon')
                            ->defaultValue('<i class="fa fa-folder"></i>')
                            ->info('Icon used for admin services if one is not provided.')
                        ->end()
```

**Comment by kunicmarko20** [comment]
Should be good now

**Comment by greg0ire** [comment]
Thanks @kunicmarko20 !

**Pull request merged**